### PR TITLE
Input Fix

### PR DESCRIPTION
**Pull request by pastaq**
Fixed a bug where menu states would replace non-menu states in certan situations rather than being pushed to the stack.

### CODE REVIEW

File: core/systems/input/input_manager.gd [hunk 1]
```diff
@@ -36,47 +36,50 @@ func _on_state_changed(from: int, to: int, _data: Dictionary):
 
 func _input(event: InputEvent) -> void:
 	var state = state_manager.current_state()
-	
 	if event.is_action_pressed("ogui_guide"):
 		if not guide_action:
 			guide_action = true
+	# Handle OSK Button combo
+	if Input.is_action_pressed("ogui_north"):
```
Comment (ShadowApex) [inline core/systems/input/input_manager.gd:43]: Should this be `ogui_west`?

Comment (pastaq) [reply core/systems/input/input_manager.gd:43]: No. West is Y. verify with `evtest`

Comment (ShadowApex) [reply core/systems/input/input_manager.gd:43]: I thought OSK was X...

Comment (pastaq) [reply core/systems/input/input_manager.gd:43]: It is...?

**Pull request merged**